python/lsst/afw/display/interface.py

```diff
@@ -80,7 +80,7 @@ def _makeDisplayImpl(display, backend, *args, **kwargs):
         try:
             _disp = importlib.import_module(dt, package="lsst.display")
             break
-        except ImportError as e:
+        except (ImportError, SystemError) as e:
```

Comment:
For example:
```
Importing '.virtualDevice' with args {'package': 'lsst.display'}
Traceback (most recent call last):
  File "tests/display.py", line 202, in <module>
    with afwDisplay.Display("test", backend=args.backend) as disp:
  File "/Volumes/G-RAID with Thunderbolt/transient/lsstsw3/build/afw/python/lsst/afw/display/interface.py", line 143, in __init__
    self._impl = _makeDisplayImpl(self, backend, *args, **kwargs)
  File "/Volumes/G-RAID with Thunderbolt/transient/lsstsw3/build/afw/python/lsst/afw/display/interface.py", line 88, in _makeDisplayImpl
    _disp = importlib.import_module(dt, **impargs)
  File "/Users/timj/work/lsstsw3/miniconda/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 981, in _gcd_import
  File "<frozen importlib._bootstrap>", line 931, in _sanity_check
SystemError: Parent module 'lsst.display' not loaded, cannot perform relative import
```
so an error triggers before it can even attempt to do the import when `lsst.display` does not exist.